jmaqs-selenium/src/main/java/com/magenic/jmaqs/selenium/AbstractLazyElement.java

```diff
@@ -585,13 +586,11 @@ private void executeEvent(Action elementAction, String caller) {
       this.getTestObject().getLogger().logMessage(MessageType.VERBOSE, "Performing lazy driver action: " + caller);
       elementAction.invoke();
     } catch (Exception e) {
-      StringBuilder messageBuilder = new StringBuilder();
-
-      messageBuilder.append("Failed to " + caller + ": " + this.userFriendlyName + System.lineSeparator());
-      messageBuilder.append("Locator: " + this.getBy() + System.lineSeparator());
-      messageBuilder.append("Because: " + e.getMessage() + System.lineSeparator());
-
-      throw new ExecutionFailedException(messageBuilder.toString(), e);
+      String messageBuilder =
```

Comment:
This does not need to be called 'messageBuilder' anymore so let's rename it to be more clear.

Reply:
changed the variable name to exceptionReason